tests/Notification/NotifierTest.php

```diff
@@ -183,7 +183,7 @@ public function testPrepare($language, $user, $changeName, $changeAvatar, $name,
 			$notification->expects($this->never())
 				->method('setLink');
 
-			$this->setExpectedException(\InvalidArgumentException::class);
+			$this->expectException(\InvalidArgumentException::class);
```

Comment:
This is only broken on PHP 5.6. I would say we should go with the deprecated setExpectedException and then move away from this once 5.6 is not supported anymore (travis ships phpunit 4.5 on PHP 5.6)